docs/reference/master-commandline-reference.md

```diff
@@ -99,6 +99,23 @@ Example:
 nfd-master -cert-file=/opt/nfd/master.crt -key-file=/opt/nfd/master.key -ca-file=/opt/nfd/ca.crt
 ```
 
+### -enable-taints
+
+The `-enable-taints` flag enables/disables node tainting feature of NFD.
+
+Default: *false*
+
+NOTE: After NFD sets a taint, already running pods that do not tolerate the taint
```

Comment:
I would re-write this to be
" Before enabling the any taints, make sure you edit nfd-worker daemonset to tolerate the taints to be created, otherwise......"
In documentation is better to recommend what to do in preparation for, not how to fix something after. (In my opinion)

Reply:
Good point. Done.